pwd_guess.py

```diff
@@ -521,6 +535,10 @@ def override_from_commandline(self, cmdline):
             answer[key] = type(getattr(self, key))(value)
         self.adict.update(answer)
 
+    def sequence_model_updates(self):
```

Comment:
Adding the tab character to the character bag will also cause it to be included as a valid character in the filtering step. This means that the input data could have passwords with the tab character. Since you are using tab as a special indicator for the start of a password, this seems like it could have undesirable effects down the line during classification. I would suggest moving it to the many to many model (maybe during preprocessing?) and adding a check that asserts that tab is not in the allowed character bag.

Reply:
The LSTM model is actually getting the tab character as an input, specifically the first character is the tab character for the many to many model. For it to be properly encoded it had to be added to the char bag.  Its the same reason why we have the '\n' character in the char bag as well